### DISCUSSION TITLE
Is it possible save the prompt in .png files?

### DISCUSSION THREAD

**Discussion by CanadianBeaver** [lllyasviel/Fooocus#3229] [Q&A] [answered]
It will be good, if the prompt and other generation settings can be stored in `.png` file and therefore can be used next time for similar generation. Thank you.

**Comment by ILikeToasters** [comment, marked as answer]
Yup. There are settings in the developer debug menu but I recommend adding the required entries to your config.txt file in the Fooocus folder so it is always enabled and you don't have to do it each time. Add these lines to your config.txt. 
```
"default_save_metadata_to_images": true,
"default_metadata_scheme": "fooocus",
```

**Reply by ILikeToasters** [reply]
If you are not familiar with editing the config.txt file then watch this: https://www.youtube.com/watch?v=jGPxvROLwUU

**Reply by ILikeToasters** [reply]
Then you can drop the image into the metadata tab in the image prompt section to get the metadata and apply it to Fooocus. You can also copy that info from your image history log as well.

**Comment by CanadianBeaver** [comment]
thank you very much for help
